config/dcos-release.config.yaml

```diff
@@ -13,6 +13,15 @@ storage:
     account_key: $AZURE_STORAGE_ACCESS_KEY
     container: dcos
     download_url: https://dcosio.azureedge.net/dcos/
+testing:
+  aws:
+    kind: aws_s3
+    bucket: downloads.dcos.io
+    object_prefix: dcos
```

Comment:
This really needs to not be the same as the production, it will create and delete a bunch of random things in s3, and if the code is wrong / has a bug (part of what this tests for), then we could end up deleting downloads.dcos.io